README.md

```diff
@@ -2,71 +2,32 @@
 
 # Project Name
 
-> Description the project.
+> Hello microverse
 
```

Comment:
There are a couple of tweaks in the README that I suggest you implement. In the `Project Name` header you need to type the project's name and in the `Description` section you need to type a description for the project.